readme.md

```diff
@@ -236,6 +236,42 @@ ow(1, 'input', ow.number.validate(value => ({
 //=> ArgumentError: Expected number `input` to be greater than 10, got 1
 ```
 
+#### message(string | fn)
+
+Provide a custom message:
+
+```ts
+ow('🌈', 'unicorn', ow.string.equals('🦄').message('Expected unicorn, got rainbow'));
+//=> ArgumentError: Expected unicorn, got rainbow
+```
+
+You can also pass in a function which receives the value as the first parameter and the label as the second parameter and is expected to return the message.
+
+```ts
+ow('🌈', ow.string.minLength(5).message((value, label) => `Expected ${label}, to have a minimum length of 5, got \`${value}\``));
+//=> ArgumentError: Expected string, to be have a minimum length of 5, got `🌈`
+```
+
+It's also possible to add a separate message per validation:
+
+```ts
+ow(
+	'1234',
+	ow.string
+		.minLength(5).message((value, label) => `Expected ${label}, to be have a minimum length of 5, got \`${value}\``)
+		.url.message('This is no url')
+);
+//=> ArgumentError: Expected string, to be have a minimum length of 5, got `1234`
+
+ow(
+	'12345',
+	ow.string
+		.minLength(5).message((value, label) => `Expected ${label}, to be have a minimum length of 5, got \`${value}\``)
+		.url.message('This is no url')
+);
+//=> ArgumentError: This is no url
+```
```

Comment:
The readme and TS doc comments should be in sync as much as possible. These examples are missing from the TS doc comments.

Reply:
Added two examples to the `message` function in `source/predicates/predicate.ts`

Reply:
Yes, if they are in the readme, they should go in index.d.ts.

Reply:
Also make sure the formatting is the same. It's not now.